packages/typescript/docs/react-native-sdk-generated.md

```diff
@@ -288,6 +288,7 @@ import { ConnectWidget } from "@mxenabled/react-native-widget-sdk"
 - Payload fields:
     - `user_guid` (`string`)
     - `session_guid` (`string`)
+    - `member_guid` (optional) (`string`)
```

Comment:
Notice the (optional) here, this is how the docs convey it's an optional field. How do you guy like this? Open to suggestions on this one, I know it's not the prettiest.

Reply:
I don't have an issue with it.